model/src/main/java/org/openremote/model/notification/Notification.java

```diff
@@ -46,6 +47,8 @@ public static class Target {
 
         protected TargetType type;
         protected String id;
+        protected String locale;
+        protected List<String> allowedLocales = new ArrayList<>();
```

Comment:
If the order is not important it might also make sense to prevent multiple occurences of the same locale:

```suggestion
        protected Set<String> allowedLocales = new HashSet<>();
```